modules/cli-installing-cli.adoc

```diff
@@ -60,7 +60,7 @@ You can install the OpenShift CLI (`oc`) binary on Linux by using the following
 .Procedure
 
 ifdef::openshift-origin[]
-. Navigate to https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/ and choose the folder for your operating system and architecture.
+. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your operating system and architecture.
```

Comment:
Using this URL seems fine to me but we should probably clarify the wording a bit because it lands on a page of OCP versions now. How about something like this?
`. Navigate to https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/ and choose the folder for your {product-title} version, operating system, and architecture.`
Seems less to maintain than using the release-specific URL and having to update the procedure with each release. :)